test/producer-consumer-spec.js

```diff
@@ -47,6 +47,29 @@ describe('producer/consumer', function () {
       });
     });
 
+
+    it('should receive message headers', () => {
+      const header =  {header1: "Header1", header2: "Header2"};
```

Comment:
minor: consider naming this `headers`, it has 2 of them and you could then use a shorthand notation on line 55

Reply:
In fact, if you take a look at the message structure:

Reply:
And judging by the data inside these `properties`, there are not exactly just headers, also some options passed to the consumer, so I would not rename it